packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -104,6 +94,11 @@ if (!canUseDOM) {
     callback: FrameCallbackType,
     options?: {timeout: number},
   ): CallbackIdType {
+    invariant(
+      typeof localSetTimeout === 'function',
+      'localSetTimeout is not a function. Please load a polyfill that defines this function.',
```

Comment:
`localSetTimeout` is a variable name so it doesn't mean anything to somebody reading this. That said, I don't think we want an invariant at all.